vscode-lean4/src/projectoperations.ts

```diff
@@ -181,6 +166,22 @@ export class ProjectOperationProvider implements Disposable {
             }
 
             await this.tryFetchingCache(lakeRunner)
+
+            const localToolchainPath: string = join(activeClient.folderUri.fsPath, 'lean-toolchain')
+            const dependencyToolchainPath: string = join(activeClient.folderUri.fsPath, 'lake-packages', dependencyChoice.name, 'lean-toolchain')
```

Comment:
Hard-coding this problematic. For one, [the Lake directory change](https://github.com/leanprover/lean4/pull/2749) will break it. Ideally, this would use the `packagesDir` setting from the root Lake manifest instead.

Reply:
Good point! I'll adjust it.